src/firecracker/src/main.rs

```diff
@@ -122,7 +119,7 @@ fn main_exitable() -> FcExitCode {
         .arg(
             Argument::new("id")
                 .takes_value(true)
-                .default_value(DEFAULT_INSTANCE_ID)
+                .default_value(logger::DEFAULT_INSTANCE_ID)
```

Comment:
Just make this a string literal in the first commit. Right now you do

```
-                .default_value(DEFAULT_INSTANCE_ID)
+                .default_value(logger::DEFAULT_INSTANCE_ID)
```

and then later:

```
-                .default_value(logger::DEFAULT_INSTANCE_ID)
+                .default_value("anonymous-instance")
```